### PR TITLE
v1alpha1/import: Set Round-Robin as the default LBScheme

### PR DESCRIPTION
**Pull request by kfirtoledo**
Set ECMP as the default LBScheme for import

### CODE REVIEW

File: config/crds/clusterlink.net_imports.yaml [hunk 1]
```diff
@@ -36,8 +36,9 @@ spec:
             description: Spec represents the attributes of the imported service.
             properties:
               lbScheme:
```
Comment (elevran) [inline config/crds/clusterlink.net_imports.yaml:38]: nit: Envoy uses `policy`, not `scheme` for this

File: config/crds/clusterlink.net_imports.yaml [hunk 2]
```diff
@@ -36,8 +36,9 @@ spec:
...
-                description: LBScheme is the load-balancing scheme to use (e.g., Random,
-                  Static, ECMP)
+                default: ecmp
+                description: LBScheme is the load-balancing scheme to use (e.g., random,
```
Comment (elevran) [inline config/crds/clusterlink.net_imports.yaml:40]: q: if the key is called `lbScheme` in line 38, why refer to `LBScheme` in the description?

File: config/crds/clusterlink.net_imports.yaml [hunk 3]
```diff
@@ -36,8 +36,9 @@ spec:
             description: Spec represents the attributes of the imported service.
             properties:
               lbScheme:
-                description: LBScheme is the load-balancing scheme to use (e.g., Random,
-                  Static, ECMP)
+                default: ecmp
```
Comment (elevran) [inline config/crds/clusterlink.net_imports.yaml:39]: what is the difference between `ecmp` and `random` modes?
I think `ecmp`  is perhaps too "low-level-network-stuff" for most users and we ought to select a simpler name (e.g., `hash`?)

**Review by elevran — COMMENTED** [review]
unclear about some of the word choices

File: config/crds/clusterlink.net_imports.yaml (hunk 3 above)
Comment (kfirtoledo) [reply config/crds/clusterlink.net_imports.yaml:39]: ECMP is implemented as a round-robin I will open an issue for that. @zivnevo

File: config/crds/clusterlink.net_imports.yaml (hunk 2 above)
Comment (kfirtoledo) [reply config/crds/clusterlink.net_imports.yaml:40]: This is an auto-generated file

File: config/crds/clusterlink.net_imports.yaml (hunk 3 above)
Comment (elevran) [reply config/crds/clusterlink.net_imports.yaml:39]: May I suggest that we have one PR that sets the default and renames it to round-robin (or `rr`) across all files and samples? I counted 9 affected files.

**Review by elevran — APPROVED** [review]
unclear about some of the word choices

**Review by zivnevo — APPROVED** [review]
ok

File: config/crds/clusterlink.net_imports.yaml (hunk 1 above)
Comment (kfirtoledo) [reply config/crds/clusterlink.net_imports.yaml:38]: @zivnevo

Comment (zivnevo) [reply config/crds/clusterlink.net_imports.yaml:38]: Back in the old days, we used Policy to encapsulate all the details the user had to set in a given execution of `gwctl` (client name, service name, scheme and default peer). Now that the scheme is set inside an Import CRD, I don't mind renaming it  as `lbPolicy`. I opened Issue #423 .

**Pull request merged**